thefuck/utils.py

```diff
@@ -108,15 +109,22 @@ def _safe(fn, fallback):
 
     tf_alias = get_alias()
     tf_entry_points = ['thefuck', 'fuck']
-
+    
+    win32 = sys.platform.startswith('win')
+    paths = os.environ.get('PATH', '')
+    paths = paths.split(';') if win32 else paths.split(':')
+    
     bins = [exe.name.decode('utf8') if six.PY2 else exe.name
-            for path in os.environ.get('PATH', '').split(':')
```

Comment:
I think we can just split on `os.pathsep` here, instead of defining `paths` above, see https://stackoverflow.com/questions/1499019/how-to-get-the-path-environment-variable-separator-in-python/1499033#1499033

Reply:
Yes, this is the elegant way, and it works fine on windows. I'll fix the lint issue and recommit it.